tests/integration/test_charm.py

```diff
@@ -6,35 +6,55 @@
 """Integration tests."""
 
 import logging
+import re
 import time
 
 import juju.action
 import pytest
 import requests
+from oauth_tools.oauth_helpers import (
+    access_application_login_page,
+    click_on_sign_in_button_by_text,
+    complete_auth_code_login,
+    deploy_identity_bundle,
+)
+from playwright.async_api import expect
 from pytest_operator.plugin import OpsTest
 
 logger = logging.getLogger(__name__)
 
+pytest_plugins = ["oauth_tools.fixtures"]
+
 
 @pytest.mark.abort_on_fail
-async def test_build_and_deploy(
-    ops_test: OpsTest, pytestconfig: pytest.Config, minio, mailcatcher
+async def test_build_and_deploy(  # pylint: disable=too-many-locals
```

Comment:
Can you add a comment to explain why you need the disable ?

Reply:
The comment I came up with is "too many local variables." Is this a little redundant?

Reply:
There's a reason why you have so many local variables. Usually this kind of linter warns needs to trigger a self-reflection on the code. Does this method should be that long ? Can't I refactor it in more readable smaller method ? Are all these variables systematically necessary everytime this method is called ?
And this will lead to some conclusion, therefore I suppose that your conclusion was that it's necessary, therefore providing part of the reasoning in the comment will avoid somebody else redoing the same exercises and potentially reaching different conclusions and change the way the code is written.

Reply:
I optimized this test case a bit, and the `# pylint: disable=too-many-locals` is no longer needed. Thanks!